lib/plurimath/math/formula.rb

```diff
@@ -585,6 +585,16 @@ def mglyph_value=(value)
         update_temp_order(value, "mglyph")
       end
 
+      def display
+        @display
+      end
+
+      def display=(value)
+        return unless value
```

Comment:
This will not allow to unset the `@display` value once it is set. May be we need an accessor here?